.github/workflows/report_quotes.yaml

```diff
@@ -0,0 +1,28 @@
+
+name: telegram quote report
+on:
+  schedule:
+    - cron: "* */10 * * *"
```

Comment:
Did you mean "0 10 * * *" ?

Reply:
Nope... i meant each 10hours, looks too much ? 🤔

Reply:
updated ! 👍🏾

Reply:
I see in this you probably wanted to say this "0 */10 * * *" 👌